### PR TITLE
Removed icons on Action Maps

### PR DESCRIPTION
**Pull request by lyndon-unity**
### Description

Removed Icons on actions maps ([ISX-1555](https://jira.unity3d.com/browse/ISX-1555))

image

### Changes made

Split action map UXML off to InputActionMapsTreeViewItem so we can have a different style for those with no icons

### Checklist

Before review:

- [X] Changelog entry added.
- [ ] Tests added/changed, if applicable.
    - Not applicable
- [ ] Docs for new/changed API's.
    - Not applicable
    -  
During merge:

- [ ] Commit message for squash-merge is prefixed with one of the list:
    - `NEW: ___`.
    - `FIX: ___`.
    - `DOCS: ___`.
    - `CHANGE: ___`.
    - `RELEASE: 1.1.0-preview.3`.

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionMapsView.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public ActionMapsView(VisualElement root, StateContainer stateContainer)
 
             m_ListView.bindItem = (element, i) =>
             {
-                var treeViewItem = (InputActionsTreeViewItem)element;
+                var treeViewItem = (InputActionMapsTreeViewItem)element;
```
Comment (jfreire-unity) [inline Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionMapsView.cs:25]: Nice 👍

**Review by jfreire-unity — APPROVED** [review]

**Comment by jfreire-unity** [comment]
@lyndon-unity just some formatting issues. After that's fixed, it should be good to be merged.

**Comment by lyndon-unity** [comment]
> @lyndon-unity just some formatting issues. After that's fixed, it should be good to be merged.

I keep missing these. Fixing and pushing now.

**Review by Billreyn — APPROVED** [review]

**Comment by lyndon-unity** [comment]
Latest commit makes sure the Edit button on the inspector opens the new UI TK window
image

**Pull request merged**